[DOCS] 9.3.1 Kibana release notes (#253548)
elastic/kibana: docs/release-notes/index.md

* Fixes links being visible on Search homepage when the user doesn't have access [#251437]({{kib-pull}}251437).
* Fixes AI Connector form fields incorrectly resetting to default values when cleared with backspace [#251095]({{kib-pull}}251095).
* Updates connector description terminology to reference "pre-configured AI connectors" [#250649]({{kib-pull}}250649).
* Fixes `runtime_mappings` being ignored or overridden in Vega specs when defined in `data[].url.body` [#253560]({{kib-pull}}253560).
* Adds datemath support to the KQL evaluator [#252840]({{kib-pull}}252840).
* Strips system-managed date fields from ingest pipelines before PUT requests [#252579]({{kib-pull}}252579).
* Handles empty results in ES|QL rule execution [#250759]({{kib-pull}}250759).
* Propagates ES|QL execution errors [#250605]({{kib-pull}}250605).
* Updates the AI Insight UI description [#250137]({{kib-pull}}250137).
* Fixes query sync on save and adds debounce for the preview chart [#249833]({{kib-pull}}249833).
* Fixes Stack Monitoring breadcrumb when in solution view [#249751]({{kib-pull}}249751).

> * Disables editing of significant event queries [#249716]({{kib-pull}}249716).

Author: Thanks - fixed in https://github.com/elastic/kibana/pull/253548/changes/837d2b3f5e4d574ad5534a2a5f81fc98a3741ab4.

% ### Fixes [kibana-X.X.X-fixes]
% *

% FEATURES, ENHANCEMENTS, FIXES
% Paste in index.md

### Features and enhancements [kibana-9.3.1-features-enhancements]

**Data ingestion and Fleet**:

> * Allow integration rollback if not all package policies are upgraded [#253646]({{kib-pull}}253646).

Reviewer: ```suggestion
* Allows integration rollback even if all package policies are not on an upgraded version [#253646]({{kib-pull}}253646).
```

**Elastic Security solution**:
For the Elastic Security 9.3.1 release information, refer to [Elastic Security Solution Release Notes](docs-content://release-notes/elastic-security/index.md).

**Machine Learning**:
* Adds a dynamic default connector to GenAI settings [#252861]({{kib-pull}}252861).
* Adds missing ES|QL commands and functions documentation for inference tasks [#249089]({{kib-pull}}249089).

**Management**:

> * Adds a concat processor [#247940]({{kib-pull}}247940).

Author: ah, right you are. I'll remove it.

```suggestion
```

### Fixes [kibana-9.3.1-fixes]

**Alerting and cases**:

> * Fixes a bug with PagerDuty where setting the Custom details field causes rules to fail [#253683]({{kib-pull}}253683).

Reviewer: ```suggestion
* Fixes a bug with PagerDuty where setting the **Custom details** field causes rules to fail [#253683]({{kib-pull}}253683).
```

> * Adds external references to checks [#253107]({{kib-pull}}253107).

Reviewer: ```suggestion
* Adds external reference IDs to the attached documents check when a case is selected [#253107]({{kib-pull}}253107).
```

* Fixes links being visible on Search Homepage when the user doesn't have access [#251437]({{kib-pull}}251437).
* Fixes AI Connector form fields incorrectly resetting to default values when cleared with backspace [#251095]({{kib-pull}}251095).
* Updates connector description terminology to reference "pre-configured AI connectors" [#250649]({{kib-pull}}250649).
* Fixes `runtime_mappings` being ignored or overridden in Vega specs when defined in `data[].url.body` [#253560]({{kib-pull}}253560).
* Fixes Beats tutorial authentication instructions for Serverless environments [#253164]({{kib-pull}}253164).
* Adds datemath support to the KQL evaluator [#252840]({{kib-pull}}252840).
* Strips system-managed date fields from ingest pipelines before PUT requests [#252579]({{kib-pull}}252579).
* Handles empty results in ES|QL rule execution [#250759]({{kib-pull}}250759).
* Propagates ES|QL execution errors [#250605]({{kib-pull}}250605).
* Updates the AI Insight UI description [#250137]({{kib-pull}}250137).
* Fixes query sync on save and adds debounce for the preview chart [#249833]({{kib-pull}}249833).

> * Fixes monitoring breadcrumbs for solution view [#249751]({{kib-pull}}249751).

Reviewer: ```suggestion
* Fixes Stack Monitoring breadcrumb when in solution view [#249751]({{kib-pull}}249751).
```
